lib/rataextra-backend.ts

```diff
@@ -382,6 +388,13 @@ export class RataExtraBackendStack extends NestedStack {
         httpRequestMethods: ['GET'],
         targetName: 'checkUserRightOnPageContents',
       },
+      {
+        lambda: getComponents,
```

Comment:
Presumably this wasn't used by frontend yet?

Reply:
Correct. Should not have any issues with this.

Reply:
So my approval stands.